messages/en-US/en-US.json

```diff
@@ -18,5 +18,92 @@
   },
   "Test": {
     "info": "You can try again later."
+  },
```

Comment:
I added these translation here, for now. We can always break up into separate `.json` files.